ports/psoc6/mphalport.c

```diff
@@ -121,6 +121,22 @@ uint8_t mp_hal_pin_name(mp_hal_pin_obj_t pin) {
     return pin;
 }
 
+void mp_hal_pin_write(mp_hal_pin_obj_t pin, uint8_t polarity) {
+    if (polarity == 1) {
+        gpio_set_value(pin);
+    } else {
+        gpio_get_value(pin);
```

Comment:
Why is this required if the return value is not used anyway ?

Reply:
Thanks! Yes It's fixed . It is gpio_clear_value ()